src/Microsoft.SqlTools.ServiceLayer/TaskServices/SqlTask.cs

```diff
@@ -96,6 +96,13 @@ public SqlTask(TaskMetadata taskMetdata, Func<SqlTask, Task<TaskResult>> testToR
             });
         }
 
+        public void Run()
+        {
+            RunAsync().ContinueWith(task =>
```

Comment:
my fault here,  please change it to Task.Run(() => RunAsync());

---
In reply to: [122303762](https://github.com/Microsoft/sqltoolsservice/pull/381#discussion_r122303762) [](ancestors = 122303762)